backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Mocks/Altinn.AccessManagement.UI.Mocks.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="Data\**">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Remove="Data\MaskinportenSchema\Delegation\appid-400\from_p50067798\to_810418532\ExpectedOutput_Default.json" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Altinn.App.Common" Version="6.0.2" />
+    <PackageReference Include="Altinn.Common.PEP" Version="1.3.0" />
+    <PackageReference Include="JWTCookieAuthentication" Version="3.0.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Altinn.AccessManagement.UI.Core\Altinn.AccessManagement.UI.Core.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Data\MaskinportenSchema\Delegation\NewFolder\NewFolder\" />
```

Comment:
NewFolder reference I assume can be removed

Reply:
Oops 😅
Nice catch!